dev/io.openliberty.security.jakartasec.3.0.internal_fat.common/src/io/openliberty/security/jakartasec/fat/utils/OpenIdContextExpectationHelpers.java

```diff
@@ -75,16 +99,37 @@ public static void getOpenIdContextIssuerExpectations(String action, Expectation
 
     }
 
-    public static void getOpenIdContextTokenTypeExpectations(String action, Expectations expectations, String requester) throws Exception {
+    public static void getOpenIdContextTokenTypeExpectations(String action, Expectations expectations, String requester, ResponseValues rspValues) throws Exception {
 
-        expectations.addExpectation(new ResponseFullExpectation(action, Constants.STRING_CONTAINS, requester + ServletMessageConstants.TOKEN_TYPE
-                                                                                                   + Constants.TOKEN_TYPE_BEARER, "Did not find the token_type set to Bearer in the OpenIdContext."));
+        if (rspValues.getTokenType() != null) {
+            expectations.addExpectation(new ResponseFullExpectation(action, Constants.STRING_CONTAINS, requester + ServletMessageConstants.TOKEN_TYPE
+                                                                                                       + Constants.TOKEN_TYPE_BEARER, "Did not find the token_type set to Bearer in the OpenIdContext."));
+        } else {
+            expectations.addExpectation(new ResponseFullExpectation(action, Constants.STRING_CONTAINS, requester + ServletMessageConstants.TOKEN_TYPE
+                                                                                                       + ServletMessageConstants.NULL, "Did not find the token_type set to Bearer in the OpenIdContext."));
```

Comment:
```suggestion
                                                                                                       + ServletMessageConstants.NULL, "Found a token_type in the OpenIdContext and should not have."));
```
Copy-paste error, I imagine.

Reply:
yup!